src/stylesheets/project/_uswds-project-settings.scss

```diff
@@ -149,33 +170,33 @@ defined in the system:
 $theme-font-definitions: (
   /*
   font-new: (
-    name: 'Font Display Name',
-    dir: 'folder-name', // relative to settings.$theme-font-path
+    name: "Font Display Name",
```

Comment:
Should be single quotes.

Reply:
Oh huh, it must not have picked up the last changes